server/@types/bapv.d.ts

```diff
@@ -11,7 +11,14 @@ export type Booker = {
 export type AddPrisonerJourney = {
   supportedPrisons: PrisonRegisterPrisonDto[]
   selectedPrison?: PrisonRegisterPrisonDto
-  prisonerDetails?: Record<string, string>
+  prisonerDetails?: {
```

Comment:
Improving typing of this, which is used to store data during the 'Add a prisoner' journey. 

Not keen on the property names that need quoting but difficult to avoid as the `govukDateInput` names fields in this way.